gcovr/tests/csv/coverage.csv

```diff
@@ -1,2 +1 @@
 filename,line_total,line_covered,line_percent,branch_total,branch_covered,branch_percent,function_total,function_covered,function_percent
-main.cpp,2,2,1.0,0,0,,1,1,1.0

```

Comment:
unexplained change

Reply:
That file was by accident under SCM. It's the output file of the run but the rm command in the makefile was commented. The reference didn't change.
I've removed the file and the comment in the makefile.

Reply:
Should we add a check if no file from SCM is changed by a test (`git status`)?

Reply:
I think no further checks are needed since the .gitignore file already excludes coverage.csv files by default.